extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -153,27 +172,50 @@ export class ConsentUI {
         this.ui_.whenBuilt().then(() => show()) :
         show();
     }
+
+    this.isVisible_ = true;
   }
 
   /**
    * Hide the UI
    */
   hide() {
+
     if (!this.ui_) {
       // Nothing to hide from;
       return;
     }
+
+    if (this.isCreatedIframe_) {
+      this.resetIframe_();
+    }
+
     if (!this.isPostPrompt_) {
       const {classList} = this.parent_;
       classList.remove('amp-active');
       classList.add('amp-hidden');
     }
+
     // Need to remove from fixed layer and add it back to update element's top
     this.baseInstance_.getViewport().removeFromFixedLayer(this.parent_);
-    toggle(this.ui_, false);
-    if (this.isCreatedIframe_) {
-      this.resetIframe_();
+    if (this.ui_) {
```

Comment:
We've already check `if(!this.us_)`. No need for the check here. 
If it is for type check, use `assertElement()` instead.